apps/ftu/js/navigation.js

```diff
@@ -308,15 +309,24 @@ exports.Navigation = {
       case '#welcome_browser':
         UIManager.mainTitle.setAttribute('data-l10n-id', 'aboutBrowser');
 
-        // Initialize the share checkbox according to the preset value
-        // of debug.performance_data.shared
-        var sharePerformance = document.getElementById('share-performance');
-        var settingName = sharePerformance.name;
+        // Initialize the radio button to preselect the basic metrics.
         var settings = navigator.mozSettings;
-        var req = settings && settings.createLock().get(settingName);
-        if (req) {
-          req.onsuccess = function() {
-            sharePerformance.checked = req.result[settingName] || false;
+        var basicMetrics = document.getElementById('metrics-basic');
```

Comment:
I'm not crazy about doing all these UI updates in Navigation, but there's no module for this panel currently so its fine for now.